File: validation/examples/hello.ex
defmodule Validation.Examples.Hello do
  def greet(name) do
    IO.puts("Hello, " <> name <> "!")
  end

  def sum(a, b) do
    a + b
  end

  def complex_match(opts \\ []) do
    opts
    |> Keyword.get(:format, :json)
    |> case do
      :json -> %{status: "ok", code: 200}
      :text -> "ok"
    end
  end
end

File: validation/examples/hello.js
function greet(name) {
  console.log('Hello, ' + name + '!');
}

const sum = (a, b) => {
  return a + b;
};

class User {
  constructor(name, role) {
    this.name = name;
    this.role = role;
  }

  isAdmin() {
    return this.role === 'admin';
  }
}
